app/frontend/src/cljs/teet/contract/contract_partners_view.cljs

```diff
@@ -448,6 +448,27 @@
   [:div {:class (<class contract-style/personnel-table-style)}
    [:h4 (str "Approvals")]])
 
+(defn- get-personnel-info-page [employee]
+  (let [key-person? (get-in employee [:company-contract-employee/key-person?])]
+    (if (true? key-person?)
+      :assign-key-person
+      :personnel-info)))
+
+(defn key-person-icon [color]
+  (case color
+    :red [Grid
+          {:container true :direction :row :justify-content :flex-start :align-items :center}
+          [:icon (icons/red-rejected)]
+          [:icon (icons/key-person teet.theme.theme-colors/red)]]
+    :green [Grid
+            {:container true :direction :row :justify-content :flex-start :align-items :center}
+            [:icon (icons/green-check)]
+            [:icon (icons/key-person teet.theme.theme-colors/green)]]
+    :grey [Grid
```

Comment:
In the codebase we've used the american spelling `gray` instead of `grey`, no particular reason for it, but I think it should be the same here.